.github/workflows/build.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         with:
           distribution: 'zulu'
           java-version: 21
-      - uses: gradle/actions/setup-gradle@v3
+      - uses: gradle/actions/setup-gradle@v4
         with:
           validate-wrappers: true
```

Comment:
This line can be removed, true is now the default value. Wanna have a follow-up PR (cause I can't update this PR)?

Reply:
Done